lib/packages/docutils/rstgen.nim

```diff
@@ -1506,7 +1506,7 @@ proc rstToHtml*(s: string, options: RstParseOptions,
   initRstGenerator(d, outHtml, config, filen, options, myFindFile,
                    rst.defaultMsgHandler)
   var dummyHasToc = false
-  var rst = rstParse(s, filen, 0, 1, dummyHasToc, options)
+  var rst = rstParse(s, filen, line=1, column=0, dummyHasToc, options)
```

Comment:
DRY: line=1, column=0 is used 3 times.
insted, define:
```nim
const LineRstDefault = 1
const ColRstDefault = 0
```
somewhere and reuse it in those places
(in future we can refactor so that nim uses those constants (renamed))